app/views/home/index.html.erb

```diff
@@ -1 +1,5 @@
+<% name = current_user.present? ? current_user.email : 'New User' %>
```

Comment:
Not merge blocking, but in general you'd want something like "Please sign in" or something like that instead of "New User".

Reply:
Thank you, I will make that change